text-icu.cabal

```diff
@@ -1,5 +1,5 @@
 name:           text-icu
-version:        0.55.1.0
+version:        0.7.0.2
```

Comment:
While there are, best I can tell after looking at the diffs, no breaking changes since 0.7.0.1, there are substantial new features (the spoofing check code and many new code points).  So, I think the PVP-conformant version number would require at least a minor-version bump.  So at least `0.7.1.0`.

Reply:
Perhaps, if the *minimum* supported version of ICU is now 55, and this was not the case before, however unlikely something older is to be still actively used, a major version bump may be justified. When the ICU library version floor is raised, even if there are no breaking changes in the Haskell API, the new package no longer builds on older systems...

Reply:
Could you make the release with any number please? I can not compile my project in Arch Linux.

Reply:
Sorry, I missed the earlier messages from January 11. Would you be happy with `0.7.1.0` @vdukhovni? I personally am in the same boat as @Anarchist666 and would just like this to be patched so my dependencies can compile again without any `extra-deps` hacks. I don't have the time, or interest to be honest, to check all ICU versions for support, but I did notice that the tests in this repository have been failing with new ICU releases for quite a while now. Someone should really take over `text-icu`.

Reply:
I think that 0.7.1.0 makes sense.  However, it is not yet clear that text-icu has an active maintainer at the moment:
* https://mail.haskell.org/pipermail/libraries/2021-January/031114.html
* https://mail.haskell.org/pipermail/libraries/2021-February/031166.html

Reply:
Got you. I've changed it manually.